pom.xml

```diff
@@ -205,6 +205,10 @@
             <groupId>org.apache.avro</groupId>
             <artifactId>avro</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
What's the reason we exclude this?

Reply:
Let's not exclude from hadoop, but instead let's udpate also the hadoop transitive dependencies, which would violate [requireUpperBoundDeps](https://maven.apache.org/enforcer/enforcer-rules/requireUpperBoundDeps.html) enforcer rule. I opened a PR that updates all vulnerable dependencies: https://github.com/snowflakedb/snowflake-ingest-java/pull/693